spec/Section 4 -- Introspection.md

```diff
@@ -331,7 +334,12 @@ Fields
 * `kind` must return `__TypeKind.INPUT_OBJECT`.
 * `name` must return a String.
 * `description` may return a String or {null}.
+* `requiresExactlyOneField` must return a Boolean
 * `inputFields`: a list of `InputValue`.
+  * if `requiresExactlyOneField` is {true}:
+    * for each {inputValue} in {inputFields}
+      * the {type} of {inputValue} must not be Non-Null
+      * the {defaultValue} of {inputValue} must be {null}
```

Comment:
I think this would be no defaultValue instead of `null` since you don't want to allow `field: Type = null`

Reply:
Spot on :+1: